docs/source/release/v6.13.0/Framework/Algorithms/Bugfixes/39125.rst

```diff
@@ -0,0 +1 @@
+- Fixes a bug in  :ref:`algm-Divide` which do not clear unit when the YUnit of lhs == rhs due to rhsBlockszie being 0.
```

Comment:
What you have is more of an implementation detail. Please reword in terms of what a user might see which has to do with ragged workspaces. Bonus points if you `:ref:Ragged_Workspace` in the release note